### DISCUSSION TITLE
Obtain the current file path and root path within typst script

### DISCUSSION THREAD

**Discussion by Roger-luo** [typst/typst#2295] [Q&A]
Is there a way of getting these two values? I'd like to automatically adjust the path based on the file location within my project.

**Comment by OlivierLangella** [comment]
Hello, I have exactly the same question to automatically produce technical documents that displays their locations in the file system (in the header for example).

PS:
I'm discovering typst and it is great ;)

**Comment by laurmaedje** [comment]
It's not possible at the moment.

**Reply by OlivierLangella** [reply]
ok thanks

**Comment by AwesomeQuest** [comment]
any updates on this? is this a planned feature?
